deliver/lib/deliver/submit_for_review.rb

```diff
@@ -22,7 +22,11 @@ def submit!(options)
       end
 
       # Finalize app submission
-      submission.complete!
+      if options[:reject_if_waiting_for_review]
```

Comment:
Will this conditional always evaluate to `true` since you set a `default_value` in `options.rb`?

If you, you can just call `submission.complete(reject_if_waiting_for_review)`

Reply:
+1